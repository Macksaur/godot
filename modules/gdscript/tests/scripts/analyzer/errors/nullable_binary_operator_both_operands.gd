func test():
    var foo: int? = null
    var bar: int? = null
    print(foo + bar) # Should error
