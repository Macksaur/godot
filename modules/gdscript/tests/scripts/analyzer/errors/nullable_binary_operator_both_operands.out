GDTEST_ANALYZER_ERROR
Both operands for the "+" operator are nullable and as such, might be unsafe to be operated together.
